test/test_helper.rb

```diff
@@ -1,3 +1,18 @@
+require 'simplecov'
```

Comment:
These changes look identical to the changes in spec_helper.rb.

Reply:
Yes, they are recommended by SimpleCov ([README](https://github.com/simplecov-ruby/simplecov#getting-started)). I can try using in only one if needed.